Tests/AsyncHTTPClientTests/HTTPClientTests.swift

```diff
@@ -720,32 +728,32 @@ class HTTPClientTests: XCTestCase {
     }
 
     func testLoopDetectionRedirectLimit() throws {
-        let httpBin = HTTPBin(ssl: true)
```

Comment:
I think this change needs to be reverted

Reply:
I could create duplicates. One with SSL and one without

Reply:
reverted